prody/proteins/interactions.py

```diff
@@ -1988,12 +1998,20 @@ def calcStatisticsInteractions(data, **kwargs):
     for element in elements:
         if element not in stats:
             values = [t[1] for t in interactions_list if t[0] == element]
-            stats[element] = {
-                "stddev": np.round(np.std(values),6),
-                "mean": np.round(np.mean(values),6),
-                "weight": np.round(float(len(values))/len(data), 6),
-                "energy": get_energy([element.split('-')[0][:3], element.split('-')[1][:3]], energy_list_type)
-            }
+            
+            try:
+                stats[element] = {
+                    "stddev": np.round(np.std(values),6),
+                    "mean": np.round(np.mean(values),6),
+                    "weight": np.round(float(len(values))/len(data), 6),
+                    "energy": get_energy([element.split('-')[0][:3], element.split('-')[1][:3]], energy_list_type)
+                }
+            except:
```

Comment:
we should put a specific type of exception. Otherwise, it can be hard to diagnose if something else happens

Reply:
I think this is the only possibility. The energy will be taken when the pair of residues exist in the txt file with pairs. I added an additional description in docs.

Reply:
Ok